packages/lib/src/vis/shared/SelectionRect.tsx

```diff
@@ -1,23 +1,44 @@
 import type { Color } from '@react-three/fiber';
+import { useMemo } from 'react';
 import type { Vector2 } from 'three';
+import { PlaneGeometry } from 'three';
 
 interface Props {
   startPoint: Vector2;
   endPoint: Vector2;
   color?: Color;
   opacity?: number;
+  borderColor?: Color;
 }
 
 function SelectionRect(props: Props) {
-  const { startPoint, endPoint, color = 'red', opacity = 0.5 } = props;
+  const {
+    startPoint,
+    endPoint,
+    color = 'red',
+    opacity = 0.5,
+    borderColor,
+  } = props;
   const width = endPoint.x - startPoint.x;
   const height = endPoint.y - startPoint.y;
 
+  const rectGeometry = useMemo(
+    () => new PlaneGeometry(Math.abs(width), Math.abs(height)),
+    [height, width]
+  );
```

Comment:
To display the border, I use [EdgesGeometry](https://threejs.org/docs/index.html#api/en/geometries/EdgesGeometry) which takes the geometry of the filled volume as arg.

That means that unfortunately, I need to create the filled rectangle geometry imperatively to have access to it declaratively below.